File: 2/CalculadoraLambda.hs
suma = (\x y-> x+y)
resta = (\x y-> x-y)
multi = (\x y-> x*y)
division = (\x y-> x/y)
main = do
 print(suma 3 6)
 print(resta 3 2)
 print(multi 2 2)
 print(division 10 2)

File: 2/CalculadoraLambda.scala


object CalculadoraLambda {

  def main(args: Array[String]) {
    
    println(suma(10, 2))
    println(resta(10,2))
    println(multi(10,2))
    println(division(10,2))

  }

  var suma = (x: Int, y: Int) => x + y
  var resta = (x: Int, y: Int) => x - y
  var multi = (x: Int, y: Int) => x * y
  var division = (x: Int, y: Int) => x / y
}